st2common/st2common/util/pack.py

```diff
@@ -112,6 +113,31 @@ def validate_config_against_schema(config_schema, config_object, config_path,
     return cleaned
 
 
+def get_pack_common_libs_path(pack_db):
+    """
+    Return the pack's common lib path. This is the path where common code for sensors
+    and actions are placed.
+
+    For example, if the pack is at /opt/stackstorm/packs/my_pack, you can place
+    common library code for actions and sensors in /opt/stackstorm/packs/my_pack/lib/.
+    This common library code is only available for python sensors and actions. The lib
+    structure also needs to follow a python convention with a __init__.py file.
+
+    :param pack_db: Pack DB model
+    :type pack_db: :class:`PackDB`
+
+    :rtype: ``str``
+    """
+    pack_dir = getattr(pack_db, 'path', None)
```

Comment:
I guess having this path here static is fine, although in other places we dynamically resolve entry point, etc.